js/views/reader_view.js

```diff
@@ -730,11 +726,6 @@ ReadiumSDK.Views.ReaderView = function(options) {
     function openPage(pageRequest, dir) {
 
         initViewForItem(pageRequest.spineItem, function(isViewChanged){
-
-            if(!isViewChanged) {
-                _currentView.setViewSettings(_viewerSettings);
-            }
-
```

Comment:
No because having this setViewSettings call here (and in the other place I removed it in) turns out to be redundant.

It is called at the proper place in initViewForItem itself:
https://github.com/readium/readium-shared-js/pull/174/files#diff-7f14747656265ce3e39fee34e732d161R255